src/lib.rs

```diff
@@ -52,7 +52,7 @@
 //! }
 //!
 //! impl ApplicationHandler for App {
-//!     fn resumed(&mut self, event_loop: &ActiveEventLoop) {
+//!     fn can_create_surfaces(&mut self, event_loop: &ActiveEventLoop) {
 //!         self.window = Some(event_loop.create_window(Window::default_attributes()).unwrap());
```

Comment:
This also advocates creating a window _after its render surface became available_?

Reply:
Yeah, same as https://github.com/rust-windowing/winit/pull/3765#discussion_r1661536569.